ega-charts/localega/values.yaml

```diff
@@ -156,7 +156,7 @@ dataedge:
   servicePort: 9059
   debug: 5058
   logpath: "/tmp/logs"
-  keystorePass: "dataedge"
+  keystorePass: "changeit"
```

Comment:
if we are going to use one, why not set it once in the `secrets` ?

Reply:
That password is set when you run `deploy-init` the default value is `changeit`.
It is needed in order to open and use the keystore.

Reply:
let us leave for each service to use different keystore passwords